src/taco-dependency-installer/elevatedInstallerProtocol.ts

```diff
@@ -23,6 +23,7 @@ module ElevatedInstallerProtocol {
         CompletedWithErrors = 12,
         NoAdminRights = 33,
         CouldNotConnect = 37,
+        RefusedPrompt = 42,
```

Comment:
Can you put in the comments a link on how you figure out which "code numbers" are unused? Also, do we have any way of knowing if either powershell or node decide in the future to start using any of these numbers?